core/src/main/res/values/strings.xml

```diff
@@ -774,4 +774,8 @@
     <!-- Widget settings -->
     <string name="widget_create_button">Create widget</string>
     <string name="widget_opacity">Opacity</string>
+
+    <string name="subscription_auto_download">Auto Download</string>
+    <string name="subscription_keep_updated">Keep Updated</string>
```

Comment:
You can re-use the `Auto Download` and `Keep Updated` strings. This has two advantages: Our translators don't have to translate them again and we can't use inconsistent strings if we use the same ones everywhere.

Reply:
okay, I'll make it change soon..

Reply:
okay, I'll make it change soon